### PR TITLE
First publication of the JSON Schema newsletter

### PR DESCRIPTION
**Pull request by benjagm**
**Summary**

This is the first edition of the JSON Schema newsletter.

### CODE REVIEW

**Review by gregsdennis — CHANGES_REQUESTED** [review]

**Comment by gregsdennis** [comment]
We're adding this to the blog?  Is there a concern that newsletters will drown out other posts?

I think it makes sense to move newsletters and even case studies out to another page.  (Maybe we can discuss elsewhere.)

File: pages/blog/posts/newsletter-april-2024.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## 🗞 Articles and News
+
+* [What even is a JSON Number?](https://blog.trl.sn/blog/what-is-a-json-number/) by Brian Terlson. A great analysis covering different JSON specifications and implementations.  
+* [Pet Store API - Just Say No!](https://apidesignmatters.substack.com/p/pet-store-api-just-say-no) by Adam Biesack exploring different alternatives to the classic Pet Store API.
+* [Validate a JSON API with Playwright & JSON Schema](https://www.mikestreety.co.uk/blog/validate-a-json-api-with-playwright-and-json-schema/) by Mike Street with a walkthrough of the call to a JSON API and its validation against a predefined JSON Schema.
+
+## 🗞 Tooling Ecosystem
+
+* This month we discovered [Better AJV Errors](https://github.com/atlassian/better-ajv-errors) by the Atlassian Team. This tool improves the readability of the JSON Schema validation errors.
```
[the remark this answers is not in the document]
Comment (benjagm) [reply pages/blog/posts/newsletter-april-2024.md:45]: Thanks Greg. I'll remove it.

**Comment by benjagm** [comment]
> We're adding this to the blog? Is there a concern that newsletters will drown out other posts?
> 
> I think it makes sense to move newsletters and even case studies out to another page. (Maybe we can discuss elsewhere.)

I agree 100%. In fact we already built the case studies page. I'll work on this next week.

Great idea! Thanks.

**Comment by benjagm** [comment]
> I think it makes sense to move newsletters and even case studies out to another page. (Maybe we can discuss elsewhere.)

Great idea, I had been thinking about it but wasn't sure. Your comment helped to decide how to move newsletters out.
We'll do something similar to the CNCF weekly: https://www.cncf.io/kubeweekly/

This is the issue:
https://github.com/json-schema-org/website/issues/634

**Comment by radhesh1** [comment]
the logo needs to changed a bit it is not visible in dark mode

**Pull request closed without merging**